### PR TITLE
Added releasenotes for beta 3

### PR DESCRIPTION
**Pull request by tbsbdr**

### CODE REVIEW

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 1]
```diff
@@ -23,6 +23,17 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 3
+
+The third beta release of Infinite Scale 2.0.0 allows clients to list the contents of "Shares" and round-offs Spaces with a no-restriction quota feature. Web is shipped with the version 5.5.0-rc.9 and focuses on upload relaibility and usability.
```
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:28]:
```suggestion
The third beta release of Infinite Scale 2.0.0 allows clients to list the contents of "Shares" and rounds-off Spaces with a no-restriction quota feature. Web is shipped with the version 5.5.0-rc.9 and focuses on upload reliability and usability. 
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -23,6 +23,17 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 3
+
+The third beta release of Infinite Scale 2.0.0 allows clients to list the contents of "Shares" and round-offs Spaces with a no-restriction quota feature. Web is shipped with the version 5.5.0-rc.9 and focuses on upload relaibility and usability. 
+
+* Improved upload UI: The upload stability has been improved, especially for usecases with many small files and complex folder hierarchies. Also the initial view of the upload UI has been simplified and shows now more details, but only on demand.
+* Improved "File already exists" dialog: When moving files to a different folder users can now choose to skip, replace or keep conflicting files. If both files should be kept, the existing filename gets extended with a numer: filename (1) filename (2) etc. Similiar to known dialogs from e.g. Windows or MacOS, users can now apply the chosen option to all conflicts.
```
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:31]:
```suggestion
* Improved "File already exists" dialog: When moving files to a different folder users can now choose to skip, replace or keep conflicting files. If both files should be kept, the existing filename gets extended with a number: filename (1) filename (2) etc. Similar to known dialogs from e.g. Windows or MacOS, users can now apply the chosen option to all conflicts. 
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 3]
```diff
@@ -23,6 +23,17 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
...
+
+* Improved upload UI: The upload stability has been improved, especially for usecases with many small files and complex folder hierarchies. Also the initial view of the upload UI has been simplified and shows now more details, but only on demand.
+* Improved "File already exists" dialog: When moving files to a different folder users can now choose to skip, replace or keep conflicting files. If both files should be kept, the existing filename gets extended with a number: filename (1) filename (2) etc. Similar to known dialogs from e.g. Windows or MacOS, users can now apply the chosen option to all conflicts. 
+* Allow listing of "Shares": Clients can now list the share jail content via `PROPFIND /dav/spaces/{sharejailid}`
+* Create Spaces without quota restiction: Space managers can now create Spaces without specific quota restrictions like "10 GB". With the quota option "No restriction" Spaces can use up all available storage entirely. This enables a more dynamic storage allocation.
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:33]: Usually we link the corresponding PR that fixed the issue in the item, as example see beta.2
Are we doing this no longer?

Comment (kulmann) [reply modules/ROOT/pages/ocis_release_notes.adoc]: done

**Pull request merged**